fluent-syntax/src/errors.js

```diff
@@ -45,6 +46,14 @@ function getErrorMessage(code, args) {
       return 'Expected literal';
     case 'E0015':
       return 'Only one variant can be marked as default (*)';
+    case 'E0016':
```

Comment:
fluent-rs uses smaller errors numbers for these because it seems to be missing a few earlier ones.

Reply:
I'd like to make error naming part of the developer productivity review after 0.7.